tests/walletFiles.spec.ts

```diff
@@ -0,0 +1,172 @@
+import os from 'os';
```

Comment:
better to wrap tests to a structure 

describe( file name / or what we're testing ) > 
 - describe(1)
   - describe(1)
 - describe(2)
 to have better output in the terminal.

and second question, will we be cover success/error cases if so, better to separate it or add [Fail]/[Success] to the description for the test

Reply:
Don't you think it will be excessive in this case?
<img width="520" alt="image" src="https://user-images.githubusercontent.com/1897530/162528228-8a19ee6c-99bb-4da3-9044-f2645c5f9165.png">

So, I agree that in case we have some complex tests, that tests not just a single module with the same name, but some meaningfully grouped business logic — that should have an additional nesting level.

Reply:
yeap looks ok.